index.html

```diff
@@ -118,7 +118,9 @@ <h2 class="small_poster_title">
                 anymore!</q
               >
             </h2>
-            <button class="button">Get Karma today</button>
+            <a href="store.html"
+              ><button class="button">Get Karma today</button></a
+            >
```

Comment:
The spacing in line 121 and 122 is odd but every time I press "save" it come out like this, could that be extension fault?

Reply:
the same problem with spacing on line 116 to 119.

Reply:
Very weird, are you using prettier?

Reply:
Yes, prettier is enabled.